rcl/test/rcl/test_init.cpp

```diff
@@ -15,11 +15,13 @@
 #include <gtest/gtest.h>
 
 #include "rcl/rcl.h"
-
+#include "rcl/arguments.h"
```

Comment:
Includes should be in alphabetical order, yah?

Reply:
Good point.